airflow/models/dagrun.py

```diff
@@ -100,6 +102,29 @@ def _creator_note(val):
         return DagRunNote(*val)
 
 
+def _get_task_instance(
```

Comment:
Why is this function needed? It seems to me this simply forward the arguments unchanged, and all caller sites can simply call `DagRun.fetch_task_instance`?

Reply:
You are correct. The only reason is this function is being used a lot across Airflow codebase, so instead of updating many calls, I would just leave the function as proxy to the private function. If you have some concerns about that I can make the modifications

Reply:
I don’t follow. You added this function in this PR, so there should be no existing calls to update.

Reply:
Oh got it! Yep, I agree